packages/skeleton-loader/package.json

```diff
@@ -32,6 +32,7 @@
   "dependencies": {
     "@leafygreen-ui/card": "^11.0.0",
     "@leafygreen-ui/emotion": "^4.0.8",
+    "@leafygreen-ui/icon": "^12.5.4",
```

Comment:
The only thing I can think of is to copy `sizeMap` into this component, but that also does not feel correct because that can cause mismatch size issues if one ever changes.